src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/SyntaxEditorBasedCodeFixProvider.FixAllProvider.cs

```diff
@@ -32,8 +33,8 @@ public SyntaxEditorBasedFixAllProvider(SyntaxEditorBasedCodeFixProvider codeFixP
 
             public sealed override async Task<CodeAction> GetFixAsync(FixAllContext fixAllContext)
```

Comment:
All changes in this file to move off the internal type and properties that are not accessible in shared layer. Everything now operates on publically available APIs.